src/main/webapp/jsp/instructorSearch.jsp

```diff
@@ -10,6 +10,7 @@
     <script type="text/javascript" src="/js/instructor.js"></script>
     <script type="text/javascript" src="/js/instructorSearch.js"></script>
     <script type="text/javascript" src="/js/additionalQuestionInfo.js"></script>
+    <script type="text/javascript" src="/js/jquery.highlight.js"></script>
```

Comment:
As described above, we have a lot of HTML comparisons to ensure minimal regressions, this causes our HTML comparisons to fail, run the tests and make the necessary changes so that they all pass